requirements.txt

```diff
@@ -1 +1,3 @@
-omego
\ No newline at end of file
+omego
+reportlab
```

Comment:
any cap for the version of reportlab and markdown?
I remember us having issue with some version and numpy.

Reply:
I just copied these over from ```requirements-server.txt```.

Reply:
since we were using ``yum install python-reportlab`` for example to install the dependency on the server and not ``pip`, the version installed via ``pip`` will be more recent so we will need to watch for possible issue

Reply:
you will need to remove ``omego``